app/views/admin/reports_rooms.html.erb

```diff
@@ -1,36 +1,81 @@
+<!-- DataTables CSS -->
+<link rel="stylesheet" type="text/css" href="//cdn.datatables.net/plug-ins/1.10.7/integration/bootstrap/3/dataTables.bootstrap.css">
+
+<!-- DataTables -->
+<script type="text/javascript" charset="utf8" src="//cdn.datatables.net/plug-ins/1.10.7/integration/bootstrap/3/dataTables.bootstrap.js"></script>
```

Comment:
We shouldn't have these CDN links in multiple files. 


Instead they should be abstracted into a partial or placed inside the `application.html`